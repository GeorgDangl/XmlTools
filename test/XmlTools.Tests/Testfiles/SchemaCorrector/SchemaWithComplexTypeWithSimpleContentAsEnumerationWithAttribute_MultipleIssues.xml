<?xml version="1.0" encoding="UTF-8"?>
<Monsters>
  <!-- Remove due to invalid enum value -->
  <Monster Resistance="Fire">Ghost</Monster>
  <!-- Correct casing -->
  <Monster Resistance="Poison">orc</Monster>
  <!-- Remove attribute due to invalid enum value -->
  <Monster Resistance="Air">Ogre</Monster>
  <!-- Correct casing in attribute -->
  <Monster Resistance="ICE">Lich</Monster>
</Monsters>

