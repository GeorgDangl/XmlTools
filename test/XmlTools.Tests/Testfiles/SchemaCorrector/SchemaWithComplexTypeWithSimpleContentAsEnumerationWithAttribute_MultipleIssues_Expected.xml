<?xml version="1.0" encoding="UTF-8"?>
<Monsters>
  <!-- Remove due to invalid enum value -->
  <!--<Monster Resistance="Fire">Ghost</Monster>-->
  <!-- Correct casing -->
  <Monster Resistance="Poison">Orc</Monster>
  <!-- Remove attribute due to invalid enum value -->
  <Monster>Ogre</Monster>
  <!-- Correct casing in attribute -->
  <Monster Resistance="Ice">Lich</Monster>
</Monsters>
